hapi-proto/src/main/proto/ScheduleService.proto

```diff
@@ -29,7 +29,26 @@ import "Response.proto";
 import "TransactionResponse.proto";
 import "Transaction.proto";
 
-/* Transactions and queries for the Schedule Service */
+/*
+Transactions and queries for the Schedule Service
+
+The Schedule Service allows transactions to be submitted without all the required signatures and provide functionality for anyone to provide the required signatures independently after a transaction was created.
```

Comment:
"The Schedule Service allows transactions to be submitted, without all the required signatures and **allows** anyone to provide them independently after a transaction has **already been** created."